go/noise/laplace_noise_test.go

```diff
@@ -172,6 +172,495 @@ func TestDeltaForThresholdLaplace(t *testing.T) {
 		}
 	}
 }
+func TestInverseCDFLaplace(t *testing.T) {
+	for _, tc := range []struct {
+		desc               string
+		lambda, prob, want float64
```

Comment:
Could you please have all these fields in their seperate line? I believe that's the idiomatic way of declaring test structs.